test/e2e/config/docker.yaml

```diff
@@ -58,25 +58,25 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1.0/metadata.yaml"
-  - name: v1.2.11 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.11/core-components.yaml"
+  - name: v1.3.6 # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
```

Comment:
maybe:

```suggestion
  - name: v1.3.6 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
```

Reply:
It's the latest published release that is automatically installed - whether or not it's supported :smile: - I think it's clearer if we leave this as is.

Reply:
@killianmuldoon I was confused because v1.4.0 declaration has the same comment (line 70), but 1.2.12 has only "supported" (line 117)